pkg/pxc/app/statefulset/node.go

```diff
@@ -265,6 +267,52 @@ func (c *Node) AppContainer(spec *api.PodSpec, secrets string, cr *api.PerconaXt
 		}...)
 	}
 
+	if cr.CompareVersionWith("1.18.0") >= 0 {
+		const LD_PRELOAD_KEY = "LD_PRELOAD"
+		const LIBJEMALLOC_PATH = "/usr/lib64/libjemalloc.so.1"
```

Comment:
The convention is to use camel case to define constants. They can also start with lowercase given that their scope is within the if clause of `1.18` version.

We can also group them like this:

	const (
		ldPreloadKey   = "LD_PRELOAD"
		libJemalloc    = "/usr/lib64/libjemalloc.so.1"
	)